maintenance/wikia/migrateImagesBetweenSwiftDC.php

```diff
@@ -193,6 +187,8 @@ private function store() {
 
 		if ( !$srcStorage->exists( $remoteFile ) ) {
 			$this->output( "\tCannot find image to sync \n" );
+			$this->imageSyncQueue->setError( self::ERROR_CANT_FIND_FILE );
```

Comment:
This is somehow misleading to me. I read it as raising an error of the entire queue while it's probably a separate item error, isn't?